pkg/api/validate/openshiftcluster_validatedynamic.go

```diff
@@ -39,7 +45,49 @@ type openShiftClusterDynamicValidator struct {
 
 	oc              *api.OpenShiftCluster
 	subscriptionDoc *api.SubscriptionDocument
-	fpAuthorizer    refreshable.Authorizer
+	fpAuthorizer    autorest.Authorizer
+}
+
+func ensureAccessTokenClaims(ctx context.Context, tokenCredential *azidentity.ClientSecretCredential, scopes []string) error {
+	var err error
+
+	timeoutCtx, cancel := context.WithTimeout(ctx, 5*time.Minute)
```

Comment:
Q: 5 min on this seems a bit excessive for some reason, but I'm most certainly wrong. What are we expecting to take 4+ minutes that would still result in success? GetToken?

Reply:
See above answer.  Old logic pasted into a new function.